src/App.test.js

```diff
@@ -2,10 +2,81 @@ import * as React from 'react'
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event'
 import App from './App';
+import { createTodo, delay } from './tests/helpers';
 
-describe('Tests for Todo App',()=>{
-    // Write tests for Todo App
-    // Add and  Delete Todo
-    it('', () => {
-      })
-})
+describe("Tests for Todo App", () => {
+  describe("Rendering", () => {
```

Comment:
Try using a more descriptive message to describe block and test suites.
Also, we shouldn't be using nested describe blocks as it can make your test suite more complex and decrease both readability and maintainability. 
Instead of multiple describe blocks create one single describe block and then multiple it/test blocks to handle different scenarios.